torch/_dynamo/variables/builder.py

```diff
@@ -1894,8 +1894,10 @@ def _clone_input(value):
             example_value = wrap_to_fake_tensor_and_record(
                 example_value, tx=tx, **kwargs
             )
-        if isinstance(example_value, torch.Tensor) and (
-            maybe_get_fake_mode(example_value) is not tx.fake_mode
+        if (
+            isinstance(example_value, torch.Tensor)
+            and example_value.device.type != "meta"
```

Comment:
I'm not completely sure if this is the right thing to do here, but it makes the dynamo tests pass